Example/PageBuilderCountdown/view/adminhtml/web/js/content-type/example-countdown/preview.js

```diff
@@ -0,0 +1,70 @@
+define([
+    'knockout',
+    'Magento_PageBuilder/js/content-type/preview'
+], function (_knockout, PreviewBase) {
+    'use strict';
+
+    function Preview(parent, config, stageId) {
+        let _this = PreviewBase.call(this, parent, config, stageId) || this;
+        this.intervalId = setInterval(this.populateCountdown, 1000, this);
+        _this.targetDate = _knockout.observable(null);
+        this.contentType.dataStore.subscribe(function (data) {
+            _this.targetDate(data.target_date);
+        });
+        _this.targetDate.subscribe(function (index) {
+            clearInterval(this.intervalId);
+            this.intervalId = setInterval(this.populateCountdown, 1000, this);
+        }.bind(this));
+    }
+
+    Preview.prototype = Object.create(PreviewBase.prototype);
+
+    /**
+     * Populate the countdown timer with values based on the target date
+     *
+     * @returns {void}
+     */
+    Preview.prototype.populateCountdown = function populateCountdown(preview) {
+        let target = '';
+        let countdownField = document.querySelector(`#${preview.contentType.id} .countdown-field`);
```

Comment:
We can't use backticks in JS (due to IE), but this is just an example so don't think we need to update it

Reply:
@cspruiell  you didn't resolve this

Reply:
thought you were saying not to fix it but, fixed! ;p